Doc/library/asyncio-eventloop.rst

```diff
@@ -1208,6 +1208,12 @@ Executing code in thread or process pools
 
       asyncio.run(main())
 
+   Note that on macOS, an entry point guard may be needed as the example would
```

Comment:
It would be better to just add `__name__ == '__main__'`. This happens on Windows too not just macOS.

Reply:
That's a good point, Windows also defaults to `spawn` so looks like it runs into the same issue as mac. Should this note still remain as an fyi, or just change the example to fix the problem and leave out the note? It's customary to have the guard in general but knowing why it fails without might still be helpful, but I don't feel too strongly either way.

Reply:
Just add `__name__ == '__main__'`, the note is not needed.